Bugfix/unr 1698 fix host url override (#1430)
spatialos/UnrealGDK: CHANGELOG.md

and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [Unreleased-`x.y.z`] - 2019-xx-xx

> - "receptionistHost <URL>" can now be used as a command line arg to change the initial connection URL for a PIE client.

Reviewer: ```suggestion
- `receptionistHost <URL>` command line argument can now be used to change the initial connection URL for a PIE client.
```

Reviewer: Also, move this into the `Features` section below.

> - "receptionistHost <URL>" can now be used as a command line arg to change the initial connection URL for a PIE client.

Reviewer: Do you have to pass this flag when launching the editor?

- The GDK no longer generates schema for all UObject subclasses. Schema generation for Actor, ActorComponent and GameplayAbility subclasses is enabled by default, other classes can be enabled using `SpatialType` UCLASS specifier, or by checking the Spatial Type checkbox on blueprints.
- Added new experimental CookAndGenerateSchemaCommandlet that generates required schema during a regular cook.
- Added the `OverrideSpatialOffloading` command line flag. This allows you to toggle offloading at launch time.

> - The initial connection from a worker will attempt to use relevant command line arguments (receptionistHost, locatorHost) to inform the connection. If these are not provided the standard connection flow will be followed. Subsiquent connections will not use command line arguments.

Reviewer 2: ```suggestion
- The initial connection from a worker will attempt to use relevant command line arguments (receptionistHost, locatorHost) to inform the connection. If these are not provided the standard connection flow will be followed. Subsequent connections will not use command line arguments.
```

Author: Done

> - The command "Open 0.0.0.0" can be used to connect a worker to using its command line arguments, simulating initial connection.

Reviewer 2: ```suggestion
- The command "Open 0.0.0.0" can be used to connect a worker using its command line arguments, simulating initial connection.
```

Author: Done

- The GDK no longer generates schema for all UObject subclasses. Schema generation for Actor, ActorComponent and GameplayAbility subclasses is enabled by default, other classes can be enabled using `SpatialType` UCLASS specifier, or by checking the Spatial Type checkbox on blueprints.
- Added new experimental CookAndGenerateSchemaCommandlet that generates required schema during a regular cook.
- Added the `OverrideSpatialOffloading` command line flag. This allows you to toggle offloading at launch time.

> - The initial connection from a worker will attempt to use relevant command line arguments (receptionistHost, locatorHost) to inform the connection. If these are not provided the standard connection flow will be followed. Subsequent  connections will not use command line arguments.

Reviewer 2: ```suggestion
- The initial connection from a worker will attempt to use relevant command line arguments (receptionistHost, locatorHost) to inform the connection. If these are not provided the standard connection flow will be followed. Subsequent connections will not use command line arguments.
```

> - The command "Open 0.0.0.0" can be used to connect a worker using its command line arguments, simulating initial connection.

Reviewer 2: Nice!
